src/App.js

```diff
@@ -1,11 +1,76 @@
 import "./App.css";
-
-
+import { Routes, Route } from "react-router-dom";
+import { Navbar } from "./components";
+import Mockman from "mockman-js";
+import RequireAuth from "./authUtils";
+import {
+  LandingRoute,
+  VideoListingRoute,
+  PlayVideo,
+  WatchLater,
+  History,
+  MyPlaylists,
+  MyPlaylist,
+  LikedVideos,
+  Login,
+  Signup,
+} from "./routes";
 function App() {
   return (
-    <div className="App">
-      <header className="App-header"></header>
-    </div>
+    <>
+      <div className="navbar-div">
+        <Navbar />
+      </div>
+
+      <Routes>
+        <Route path="/" element={<LandingRoute />} />
+        <Route path="/videos/:country" element={<VideoListingRoute />} />
+        <Route path="/play-videos" element={<PlayVideo />} />
+        <Route
+          path="/watch-later"
+          element={
+            <RequireAuth>
+              <WatchLater />
+            </RequireAuth>
+          }
+        />

```

Comment:
Make component called requiresauth and then wrap that single component on all the private routes.
`import { useSelector } from "react-redux";
import { Navigate, Outlet } from "react-router-dom";
const PrivateRoutes = () => {
  const token = useSelector((state) => state.auth.token);
  return token ? <Outlet /> : <Navigate to="/login" replace />;
};

export { PrivateRoutes };
`